templates/writing-task-topics.rst

```diff
@@ -0,0 +1,240 @@
+:orphan: true
+	 
+.. Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type, with learnings from the 4 sfp pages built in branch DM-8717
+
+###################
+Writing Task Topics
+###################
+
+This page describes how to document science pipelines tasks for the
```

Comment:
Huh.. guess you mean sentences should have no line breaks (including those inserted by editors when i do a left justify cmd).  Ok, will do this.